e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -137,7 +137,7 @@ export function visitQuestion(id) {
  *
  * @param {number} dashboard_id
  */
-export function visitDashboard(dashboard_id) {
+export function visitDashboard(dashboard_id, { params = {} } = {}) {
```

Comment:
this was added in #29391 but didn't get backported, so it's going in with this PR